src/containers/App.jsx

```diff
@@ -158,4 +172,4 @@ App.contextTypes = {
 withRouter is necessary here to deal with redux
 https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/guides/blocked-updates.md
 */
-export default withClient(withRouter(appEntryPoint(App)))
+export default withClient(withRouter(withCustomWallpaper(appEntryPoint(App))))
```

Comment:
we should add the test we removed here https://github.com/cozy/cozy-home/pull/1614/commits/2dfa76986feb7db0f07d740881668eb58fd100c4#diff-3dd68e305272351a15051d9597703d523d88a2bcb1ef8631ceee14864dca7bd5L42-L55